### ISSUE TITLE
Concordances, when displayed in full, can cause chant detail pages to be _very_ long

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#522]
Not of vital importance, but yes, the concordances can cause the page to be quite long. Mostly an issue when printing; see the 6-page document below, 5 and a half pages of which are concordances:

[Aspiciens a longe ecce video | Cantus Manuscript Database.pdf](https://github.com/DDMAL/CantusDB/files/10412864/Aspiciens.a.longe.ecce.video.Cantus.Manuscript.Database.pdf)

could be solved by wrapping the concordances in `<details>` tags?

**Comment by annamorphism** [comment]
this would make concordances behave more like they do in OldCantus, yes? where there's a button to show them all and they magically appear?

**Comment by jacobdgm** [comment]
Yes, the behavior would be similar. A difference if we used `<details>` tags is that all the concordances would be loaded when the page loaded - `<details>` only affects whether or not they're displayed. We could choose whether they start out displayed or hidden on page load. On OldCantus, my understanding is that the table of concordances is not fetched from CantusIndex until the user clicks on "Display Concordances".

**Comment by jacobdgm** [comment]
leaving a note here for future me: in `views/views.py`, we have an `ajax_concordance_list` function. It used to only fetch concordances from CD - now that we fetch concordances from CI, I don't think it's used for anything. It would be worth going back to look at whether it can be reworked to pull data from CI and get it to be loaded and displayed only when a user clicks on "display concordances".

**Comment by jacobdgm** [comment]
also _extremely_ long for any Gloria Patri (cid 909000) chant...

**Comment by jacobdgm** [comment]
In a video call yesterday, Debra requested that the concordances begin collapsed, and that by clicking, one be able to view all the concordances. Bumping this from priority: low to medium.

If we want to really optimize page load times (not a problem for the chant detail page in my experience), we could wait to fetch these concordances until the user clicks on "view all concordances", and then we add the concordances to the page via javascript. Or if we want to use a python-only solution, we can fetch the concordances right away, and wrap the concordances in `<details>` tags.

**Comment by annamorphism** [comment]
the concordances collapse these days, so this issue is no longer necessary.
